src/victory-primitives/clip-path.js

```diff
@@ -28,28 +29,25 @@ export default class ClipPath extends React.Component {
     translateY: PropTypes.number
   };
 
+  static defaultProps = {
+    translateX: 0,
+    translateY: 0
+  }
+
   componentWillMount() {
-    const { x, y, width, height } = this.calculateAttributes(this.props);
-    this.x = x;
-    this.y = y;
-    this.width = width;
-    this.height = height;
+    merge(this, this.calculateAttributes(this.props));
```

Comment:
True. `Object.assign` would work in this case - I just wanted to aim for something that could be consistently used across the board. I could easily see us adding a nested value (such as `style` on some of the other primitives) and then fall prey to a gotcha we didn't expect.